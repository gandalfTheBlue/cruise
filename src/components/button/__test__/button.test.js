import React from 'react'
import renderer from 'react-test-renderer'

import Button from '../button'

it('renders correctly button theme default', () => {
  const tree = renderer.create(<Button theme='default' />).toJSON()
  expect(tree).toMatchSnapshot()
})

it('renders correctly button theme dark', () => {
  const tree = renderer.create(<Button theme='dark' />).toJSON()
  expect(tree).toMatchSnapshot()
})
