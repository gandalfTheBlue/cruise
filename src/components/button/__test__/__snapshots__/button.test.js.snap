// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly button theme dark 1`] = `
<button
  className="cruise-btn cruise-btn-dark"
/>
`;

exports[`renders correctly button theme default 1`] = `
<button
  className="cruise-btn cruise-btn-default"
/>
`;
